pkg/api/persistentvolume/util_test.go

```diff
@@ -87,6 +88,7 @@ func TestDropDisabledFields(t *testing.T) {
 
 	for name, tc := range tests {
 		t.Run(name, func(t *testing.T) {
+			featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.34"))
```

Comment:
This is ok now because we only have VAC feature in these tests, but if we add a new feature in a future release, are we going to have to split the tests into per-feature or per-emulation version tests?